#!/usr/bin/python3


# function that raises a name exception whit a message
def raise_exception_msg(message=""):
    raise NameError(message)
